### PR TITLE
Add storageClassName label to persistent volume claim manifest in doc…

### LINKED ISSUE #1362: Adding storageClassName to persistent volume claim manifest
Hi

I have a persistent volume claim manifest file created from Kompose convert.  I noticed the manifest file is missing a storageClassName attribute that I will like to include.  Also, there is currently no kompose label that supports adding a storageClassName to the generated manifest file.  Could there be any way I could instruct Kompose convert to add storageClassName to the spec of a PVC instead of manually editing the file after it has been generated?

Kompose convert will generate the spec below in a PVC:
```
spec:
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 1Gi
```

But I will like the ability to configure it to generate the below:

```
spec:
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 1Gi
  storageClassName: 'custom-storage-class-name'
```

### PR DESCRIPTION
**Pull request by Wusuluren**
…kompose v3 to Kubernetes (fix #1362)

### CODE REVIEW

**Comment by Wusuluren** [comment]
#1362

File: pkg/loader/compose/v3.go [hunk 1]
```diff
@@ -654,20 +655,22 @@ func handleV3Volume(komposeObject *kobject.KomposeObject, volumes *map[string]ty
...
+	size, selector, storageClassName := "", "", ""
 
 	if volume, ok := (*volumes)[name]; ok {
 		for key, value := range volume.Labels {
 			if key == "kompose.volume.size" {
 				size = value
 			} else if key == "kompose.volume.selector" {
 				selector = value
+			} else if key == "kompose.volume.storageClassName" {
+				storageClassName = value
```
Comment (hangyan) [inline pkg/loader/compose/v3.go:668]: Please read the user guide and compare to other kompose labels, I think the labels do not use CamelCase

File: pkg/transformer/kubernetes/kubernetes.go [hunk 2]
```diff
@@ -858,7 +859,7 @@ func (k *Kubernetes) ConfigVolumes(name string, service kobject.ServiceConfig) (
 					}
 				}
 
-				createdPVC, err := k.CreatePVC(volumeName, volume.Mode, defaultSize, volume.SelectorValue)
+				createdPVC, err := k.CreatePVC(volumeName, volume.Mode, defaultSize, volume.SelectorValue, volume.StorageClassName)
```
Comment (hangyan) [inline pkg/transformer/kubernetes/kubernetes.go:862]: Missing docs update ,unit test and integreated test. Please check the github ci workflow

Comment (Wusuluren) [reply pkg/transformer/kubernetes/kubernetes.go:862]: add test file

File: pkg/loader/compose/v3.go (hunk 1 above)
Comment (Wusuluren) [reply pkg/loader/compose/v3.go:668]: updated

File: script/test/cmd/tests.sh [hunk 3]
```diff
@@ -475,6 +475,11 @@ cmd="kompose -f $KOMPOSE_ROOT/script/test/fixtures/image-pull-policy/compose-fil
 sed -e "s;%VERSION%;$version;g" -e "s;%CMD%;$cmd;g"  $KOMPOSE_ROOT/script/test/fixtures/image-pull-policy/provider-files/kubernetes-v3-image-pull-policy.json > /tmp/output-k8s.json
```
[the remark this answers is not in the document]
Comment (Wusuluren) [reply script/test/cmd/tests.sh]: updated

File: docs/user-guide.md [hunk 4]
```diff
@@ -273,6 +274,19 @@ services:
       - db-data:/var/lib/postgresql/data
 ```
 
+- `kompose.volume.storage-class-name` defines the requests storage's size in the PersistentVolumeClaim, or you can use command line parameter `--pvc-request-size`.
```
[the remark this answers is not in the document]
Comment (Wusuluren) [reply docs/user-guide.md]: updated

File: docs/user-guide.md [hunk 5]
```diff
@@ -286,6 +287,18 @@ services:
...
+
+```yaml
+version: '3'
+volumes:
+  volume:
+    labels:
+      kompose.volume.storage-class-name: custom-storage-class-name
+```
+
 - `kompose.controller.type` defines which controller type should convert for this service
```
Comment (hangyan) [inline docs/user-guide.md:302]: docker-compose does not support lables in volumes filed. Please read the document refence: https://docs.docker.com/compose/compose-file/compose-file-v3/

Comment (Wusuluren) [reply docs/user-guide.md:305]: change lables to service field

**Pull request merged**